packages/contracts/contracts/foundry/SphinxUtils.sol

```diff
@@ -864,13 +864,15 @@ contract SphinxUtils is SphinxConstants, StdUtils {
 
     function getNumRootAccountAccesses(
         Vm.AccountAccess[] memory _accesses,
-        address _safeAddress
+        address _safeAddress,
```

Comment:
There's only a single change to this file: we now check the `chainId` field of the account accesses